src/components/submission/field-dropdown.vue

```diff
@@ -302,6 +302,8 @@ export default {
     },
     "disabled": "Cannot select more than 100 columns.",
     "action": {
+      // This text is shown in a dropdown that allows the user to select which
+      // columns to display in a table.
```

Comment:
this doesn't seem related to the Italian translation but i'm assuming you saw other translation things (comments) you wanted to fix up.